app/src/main/java/org/sopt/at/home/HomeViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.at.home
+
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+
+class HomeViewModel: ViewModel() {
+    val tabItems = mutableListOf("DRAMA", "VARIETY", "MOVIE", "SPORTS", "ANIMATION")
+    var selectedTabIndex = mutableStateOf(0)
+
+    fun selectTab(index:Int){
+        selectedTabIndex.value = index
+    }
+}
```

Comment:
지금 `tabItems`를 `mutableListOf`로, `selectedTabIndex`를 `mutableStateOf`로 관리하고 계신데, 
ViewModel 내부 상태는 `StateFlow`나 `MutableStateFlow`를 사용하는 것을 권장합니다!
왜 mutableListOf나 mutableStateOf를 쓰지 않는게 좋으냐 ~ mutableListOf는 단순한 가변 리스트라 변경 시 View에서는 변화를 감지할 수 없고, mutableStateOf는 composable에서 작동하는데, viewmodel은 composable에 종속되지 않고 별도의 레이어로 상태를 관리하는게 바람직해요.

또, StateFlow를 사용하면 데이터 흐름이 명확해지고 예측 가능한 상태 관리가 가능해지며, 변경이 발생할 때만 신호를 주기 때문에, 불필요한 recomposition을 줄여 성능상으로 더 좋습니다!
```suggestion
class HomeViewModel : ViewModel() {
    private val _tabItems = MutableStateFlow(listOf("DRAMA", "VARIETY", "MOVIE", "SPORTS", "ANIMATION"))
    val tabItems: StateFlow<List<String>> = _tabItems.asStateFlow()

    private val _selectedTabIndex = MutableStateFlow(0)
    val selectedTabIndex: StateFlow<Int> = _selectedTabIndex.asStateFlow()

    fun selectTab(index: Int) {
        _selectedTabIndex.value = index
    }
}
```
수정하면 이런 느낌으로 수정될 수 있겠네용
-> 이러면 View 단에서는 ViewModel이 제공하는 StateFlow만 알면 되니까 굿입니다

Reply:
+) 지금은 카테고리만 있는데, 현재 홈에서는 배너, top20, 현재 방영 중인 리스트가 필요하니 모두 추가해서 리팩토링해보시면 좋을 것 같아요 ~~